src/server/endpoints/covidcast_utils/model.py

```diff
@@ -262,21 +262,21 @@ def count_signal_time_types(source_signals: List[SourceSignalPair]) -> Tuple[int
     return daily, weekly
 
 
-def create_source_signal_alias_mapper(source_signals: List[SourceSignalPair]) -> Tuple[List[SourceSignalPair], Optional[Callable[[str, str], str]]]:
+def create_source_signal_alias_mapper(source_signals: List[SourceSignalSet]) -> Tuple[List[SourceSignalSet], Optional[Callable[[str, str], str]]]:
     alias_to_data_sources: Dict[str, List[DataSource]] = {}
-    transformed_pairs: List[SourceSignalPair] = []
-    for pair in source_signals:
-        source = data_source_by_id.get(pair.source)
+    transformed_sets: List[SourceSignalSet] = []
+    for set in source_signals:
```

Comment:
similar to comment above; perhaps use `ssset` here?

Reply:
That name feelsss very Pythonic... 🐍